widget/gridwrap.go

```diff
@@ -459,7 +511,13 @@ func (l *gridWrapLayout) setupGridItem(li *gridWrapItem, id GridWrapItemID, focu
 		f(id, li.child)
 	}
 	li.onTapped = func() {
+		canvas := fyne.CurrentApp().Driver().CanvasForObject(l.list)
+		if canvas != nil {
+			canvas.Focus(l.list)
+		}
+
 		l.list.Select(id)
+		l.list.currentFocus = id
```

Comment:
Need Refresh() call here to update the focus indicator background rectangles. If you tap a new cell the focus indicator doesn't disappear from the previously-focused cell until you mouse over it.